lib/assets/javascripts/utils/links.js

```diff
@@ -98,6 +112,14 @@ $(() => {
     const max = target.closest('.links').attr('data-max-number-links');
     target.text(convertToText(max).toLowerCase());
   });
+
+  $('.links').on('blur', 'input', (e) => {
+    toggleValidations($(e.target).closest('.link'));
+  });
+
+  $('.links').each((idx, el) => {
```

Comment:
I don't think this is needed. If you look at places that use the link functionality, (e.g. https://github.com/DMPRoadmap/roadmap/blob/master/lib/assets/javascripts/views/org_admin/templates/new.js#L7) triggers the enableValidation for that isolated context when the page is load instead of this generic handler that you have added and can be problematic for future tweaks.